press/api/account.py

```diff
@@ -64,24 +64,54 @@ def signup(email, product=None, referrer=None):
 	frappe.set_user(current_user)
 	if account_request:
 		return account_request.name
-	return None
 
 	return None
 
 
 @frappe.whitelist(allow_guest=True)
 @rate_limit(limit=5, seconds=60 * 60)
```

Comment:
Rate limiting is IP based so this is still prone to multiple attempts from different IPs. We should atleast expire the OTP within say 5 minutes?